docs/integrations/destinations/oracle-migrations.md

```diff
@@ -0,0 +1,61 @@
+# Oracle Migration Guide
+
+## Upgrading to 1.0.0
+
+This version removes the option to use "normalization" with Oracle. It also changes
+the schema and database of Airbyte's "raw" tables to be compatible with the new
+[Destinations V2](https://docs.airbyte.com/release_notes/upgrading_to_destinations_v2/#what-is-destinations-v2)
+format. These changes will likely require updates to downstream dbt / SQL models. After this update,
+Airbyte will only produce the ‘raw’ v2 tables, which store all content in JSON. These changes remove
+the ability to do deduplicated syncs with Oracle.  
+
+This upgrade will ignore any existing raw tables and will not migrate any data to the new schema.
+For each stream, you could perform the following query to migrate the data from the old raw table
```

Comment:
For each stream, you should perform the following query to migrate your data from the old raw table to the new raw table: